vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -61,6 +62,7 @@ public Dialog() {
         getElement().appendChild(template);
 
         container = new Element("div");
+        container.setAttribute("class", "draggable");
```

Comment:
nit: `container.getClassList().add("draggable");` might be cleaner

Reply:
Right. I was looking for something like `addClassName` but didn't find this one . Changed.